Okta.AspNet.Abstractions/OktaWebApiOptionsValidator.cs

```diff
@@ -3,10 +3,18 @@
 // Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
 // </copyright>
 
+using System;
+
 namespace Okta.AspNet.Abstractions
 {
     public sealed class OktaWebApiOptionsValidator : OktaWebOptionsValidator<OktaWebApiOptions>
     {
-        // No additional validation required here (yet)
+        protected override void ValidateInternal(OktaWebApiOptions options)
+        {
+            if (string.IsNullOrEmpty(options.AuthorizationServerId))
+            {
+                throw new ArgumentException("The Org Authorization Server is not supported.", nameof(options.AuthorizationServerId));
```

Comment:
Am I understanding correctly that if no value is specified for AuthorizationServerId the assumption is the Org Authorization Server would be used, but that is not supported?

Reply:
That's correct. The AS ID determines the [issuer URL](https://github.com/okta/okta-aspnet/blob/master/Okta.AspNet.Abstractions/UrlHelper.cs#L14).